packages/legacy/core/App/components/misc/InactivityWrapper.tsx

```diff
@@ -0,0 +1,121 @@
+import { useAgent } from '@credo-ts/react-hooks'
+import React, { PropsWithChildren, useEffect, useRef } from 'react'
+import { AppState, PanResponder, View } from 'react-native'
+import { useAuth } from '../../contexts/auth'
+import { useStore } from '../../contexts/store'
+import { DispatchAction } from '../../contexts/reducers/store'
+import { TOKENS, useServices } from '../../container-api'
+
+export enum LockOutTime {
+  OneMinute = 1,
+  ThreeMinutes = 3,
+  FiveMinutes = 1,
+  Never = 0,
+}
+
+interface InactivityWrapperProps {
```

Comment:
Type or Interface? I tend to favour `type` but maybe interface is better?

Re: https://www.youtube.com/watch?v=zM9UPcIyyhQ

Reply:
I have no strong feelings towards either, it is a type for now.